tools/flowc/flowc_usage.flow

```diff
@@ -54,6 +54,11 @@ printUsage(config) {
 	prih(help,"      java-main=<name>          Specify the java main class for the project. By default it is a name of a programm");
 	prih(help,"      java-incremental=1        Switch on incremental java target generation");
 	prih(help,"      java-library=id1,id2,...  Build a java library with id1, id2, ... interface");
+	prih(help,"      java-builder=<builder>    Use a specified build system. Now available: gradle or none");
```

Comment:
What is the default?

Reply:
Default is 'none' i.e. default built-in low-level jar builder. Comment is appended with this note.